src/main/java/study/step4/InputView.java

```diff
@@ -0,0 +1,22 @@
+package study.step4;
+
+import java.util.Scanner;
+
+public class InputView {
+    public static final Scanner scanner = new Scanner(System.in);
```

Comment:
상수로 선언하신 거라면 컨벤션에 따라 이름은 대문자로 작성해주세요.
```suggestion
    public static final Scanner SCANNER = new Scanner(System.in);
```

Reply:
네, 상수선언 = 대문자, 인지하고 변경하도록 할게요~.